src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+// #region Stars
```

Comment:
remove all comments not just in this place, but throughout your code

Reply:
Done! Thank you!)